src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,1991 @@
+        private static event EventHandler<AllRecognitionResultEventArgs> _allRecognitionResult;
+        private static VcMgrPreResultCallback _preResultDelegate;
+        private static VcMgrSpecificEngineResultCallback _specificEngineResultDelegate;
+        private static VcMgrBeginSpeechDetectedCallback _beginSpeechDetectedDelegate;
+        private static VcMgrDialogRequestCallback _conversationRequestDelegate;
+        private static VcMgrPrivateDataSetCallback _privateDataSetDelegate;
+        private static VcMgrPrivateDataRequestedCallback _privateDataRequestedDelegate;
+        private static VcMgrFeedbackAudioFormatCallback _feedbackAudioFormatDelegate;
+        private static VcMgrFeedbackStreamingCallback _feedbackStreamingDelegate;
+        private static VcMgrVcTtsStreamingCallback _vcTtsStreamingDelegate;
+        private static VcMgrResultCallback s_resultDelegate;
+        private static VcMgrCurrentLanguageChangedCallback _languageDelegate;
+        private static VcMgrStateChangedCallback _stateDelegate;
+        private static VcMgrServiceStateChangedCallback _serviceStateDelegate;
+        private static VcMgrErrorCallback _errorDelegate;
+        private static SelectRecognizedCommandsDelegate _recognizedCommandsDelegate;
+
+        private static VcMgrAllResultCallback _recognizedCommandsSelectionCallback = (RecognizedResult result, IntPtr vcCmdList, IntPtr recognizedText, IntPtr msg, IntPtr userData) =>
+        {
+            string recognizedString = Marshal.PtrToStringAnsi(recognizedText);
+            string msgString = Marshal.PtrToStringAnsi(msg);
+
+            if (_allRecognitionResult != null)
+            {
+                AllRecognitionResultEventArgs args = new AllRecognitionResultEventArgs(result, recognizedString, msgString);
+                _allRecognitionResult?.Invoke(null, args);
+                if (result == RecognizedResult.Rejected)
+                    return true;
+            }
+
+            if (_recognizedCommandsDelegate != null && result == RecognizedResult.Success)
+            {
+                VoiceCommandsGroup cmdList = new VoiceCommandsGroup(new SafeCommandListHandle(vcCmdList));
+                IEnumerable<VoiceCommand> selectedList = _recognizedCommandsDelegate.Invoke(cmdList.Commands, recognizedString, msgString);
+                if (selectedList == null)
+                    return true;
+
+                VoiceCommandsGroup resultList = new VoiceCommandsGroup();
+                foreach (VoiceCommand command in selectedList)
+                {
+                    resultList.Commands.Add(command);
+                }
+                ErrorCode error = VcMgrSetSelectedResults(resultList._handle);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "SetSelectedResults Failed with error " + error);
+                    throw ExceptionFactory.CreateException(error);
```

Comment:
Do not throw exception on native callback handler

Reply:
Fixed.